libraries/AP_UAVCAN/AP_UAVCAN.h

```diff
@@ -203,6 +223,11 @@ class AP_UAVCAN {
 
     AP_HAL::CANManager* _parent_can_mgr;
 
+    AP_Int16 _broadcast_bm;
+    AP_Int8 _broadcast_fix_rate;
```

Comment:
128Hz max seems fine now, but I'm thinking that we may actually want to pump this up to 400Hz in the future if the CAN bus gets used as a sensor-hub thing. Would be a good idea to make these rates AP_Int16 just in case.

Reply:
Fixed